sql/core/src/test/scala/org/apache/spark/sql/errors/QueryParsingErrorsSuite.scala

```diff
@@ -124,4 +124,49 @@ class QueryParsingErrorsSuite extends QueryTest with SharedSparkSession {
           |--------------^^^
           |""".stripMargin)
   }
+
+  test("SPARK:38104: INVALID_SQL_SYNTAX - redefine window") {
+    validateParsingError(
+      sqlText = "SELECT min(a) OVER win FROM t1 WINDOW win AS win, win AS win2",
+      errorClass = "INVALID_SQL_SYNTAX",
+      sqlState = "42000",
+      message =
+        """
+          |Invalid SQL syntax: The definition of window 'win' is repetitive.(line 1, pos 31)
+          |
+          |== SQL ==
+          |SELECT min(a) OVER win FROM t1 WINDOW win AS win, win AS win2
+          |-------------------------------^^^
+          |""".stripMargin)
+  }
+
+  test("SPARK:38104: INVALID_SQL_SYNTAX- invalid window reference") {
```

Comment:
Please, follow the existing convention for test titles in the file:
```suggestion
  test("INVALID_SQL_SYNTAX: invalid window reference") {
```

Reply:
fixed.